frontend/src/components/public_group/PublicGroupHero.js

```diff
@@ -7,22 +7,45 @@ export default class PublicGroupHero extends React.Component {
 
   renderHeroStatistics()
   {
-    const { group } = this.props;
-    if (!group.backers.length) return;
-    const backerCount = filterCollection(group.backers, {tier: 'backer'}).length;
-    const sponsorCount = filterCollection(group.backers, {tier: 'sponsor'}).length;
+    const { group, i18n } = this.props;
     const yearlyIncome = group.yearlyIncome / 100;
     const formattedYearlyIncome = yearlyIncome && formatCurrency(yearlyIncome, group.currency, { compact: true, precision: 0 });
-    return (
+
+    let tierCountString;
+    const tierCountStringArray = group.tiers.slice()
+    .sort((tierA, tierB) => tierB.range[0] - tierA.range[0])
+    .map(tier => {
+      const count = filterCollection(group.backers, {tier: tier.name}).length;
+      return (count) ? `${count} ${count === 1 ? tier.name : `${tier.name}s`}` : '';
+    })
+    .filter(x => x);
+
+    if (tierCountStringArray.length > 1)
+    {
+      if (tierCountStringArray.length > 2)
+      {
+        const lastCountString = tierCountStringArray.pop();
+        tierCountString = `${tierCountStringArray.join(', ')} ${i18n.getString('and')} ${lastCountString}`
```

Comment:
I agree 100%, however I don't see how making them sentences will help. The sentence here is dynamic, it could be one tier, or more (which will require the 'and'), all this after the "we have" words. 

The i18n sentence would have to be itself a rendering template string for something like `util.format()`, where you add parameters and depending on the languages own grammatical rules it places the data (a number in this case) in the right place.